lizard_auth_server/models.py

```diff
@@ -746,3 +740,136 @@ def as_dict(self):
             "organisation": self.organisation.as_dict(),
             "role": self.role.as_dict()
         }
+
+
+# New models
+# ##########
+
+class Profile(models.Model):
+    """Replacement for UserProfile."""
+    user = models.OneToOneField(
+        User,
+        verbose_name=_('user'),
+        related_name='profile')
+    company = models.ForeignKey(
+        'Company',
+        verbose_name=_('company'),
+        blank=True,
+        null=True)
+    created_at = models.DateTimeField(
+        verbose_name=_('created on'),
+        auto_now_add=True)
+    updated_at = models.DateTimeField(
+        verbose_name=_('updated on'),
+        auto_now=True)
+
+    def __str__(self):
+        return self.user.username
+
+    class Meta:
+        verbose_name = _("user profile")
+        verbose_name_plural = _("user profiles")
+
+    @property
```

Comment:
Al deze `@property` methodes, hebben we die wel echt nodig? Scheelt wat tikwerk, maar het is misschien wel zo duidelijk om het met `profile.user.username` "rechtstreeks" te doen.

Indien niet: al deze property methodes moeten getest worden :-)